### PR TITLE
LPS-125214 Make sure the header and the body are set to the initial width

### PR DESCRIPTION
**Pull request by holatuwol**
https://issues.liferay.com/browse/LPS-125214

## Steps to reproduce

1. Go to Content & Data -> Web Content and add a basic new web content
2. Choose to add an image to the web content and upload a new image using the file selector.
3. Select the info button for the image to open it, then again to close it

**Expected Results**:
You get a sliding behavior (all text slides into view, as originally formatted)

**Actual Results**:
You get an accordion-like behavior, where at lower widths, the text wraps extremely early, and only fixes itself at higher widths.

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js [hunk 1]
```diff
@@ -463,10 +463,25 @@ SideNavigation.prototype = {
 		const options = instance.options;
 
 		const container = document.querySelector(options.container);
+		const body = container.querySelector('.sidebar-body')
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js:466]: Apparently missing a semicolon here

**Comment by jbalsas** [comment]
Hey @holatuwol, there seems to be some formatting issue in the `side_navigation.es.js` file:

> [exec] \$ liferay-npm-scripts check
> [exec] apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js: BAD
> [exec] Prettier checked 1 file, 1 file has problems

I'm guessing it's this [missing semicolon](https://github.com/liferay-frontend/liferay-portal/pull/651/files#r547141962) but it could be also something else (sorting of grouped const seems a bit off too).

When running on CI, we only show the summary and not the whole list of errors. We also don't attempt to autofix them which should apply in this case. You can have more information running `yarn format` or `gradlew formatSource -a` in the module directly. That should also correct auto-fixable issues.

**Comment by holatuwol** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
Behaviour feels improved, so let's get this in!

/cc @liferay-lima, @ambrinchaudhary, testing this I noticed there's probably a wrong "container" in the toolbar in the item selector forcing the buttons to the middle rather than taking the whole screen. Just letting you know so you're aware. Feel free to reach out if you want us to take a look!

Screen Shot 2020-12-22 at 14 49 19

/cc @edalgrin, @marcoscv-work, @pat270, just FYI so we keep these in mind...

**Comment by jbalsas** [comment]
ci:forward

**Comment by edalgrin** [comment]
> Behaviour feels improved, so let's get this in!
> 
> /cc @liferay-lima, @ambrinchaudhary, testing this I noticed there's probably a wrong "container" in the toolbar in the item selector forcing the buttons to the middle rather than taking the whole screen. Just letting you know so you're aware. Feel free to reach out if you want us to take a look!
> 
> Screen Shot 2020-12-22 at 14 49 19
> 
> /cc @edalgrin, @marcoscv-work, @pat270, just FYI so we keep these in mind...

@jbalsas isn't this the same of https://liferay.slack.com/archives/CNBG06JS3/p1608258406461200 ?

**Comment by jbalsas** [comment]
> @jbalsas isn't this the same of https://liferay.slack.com/archives/CNBG06JS3/p1608258406461200 ?

Indeed! Hadn't seen the whole thing!

**Pull request closed without merging**